root/usr/lib/systemd/system/nsdc.service

```diff
@@ -14,6 +14,7 @@ Environment=OPTIONS=
 Environment=BRIDGE=brdef
 EnvironmentFile=-/etc/sysconfig/nsdc
 ExecStart=/usr/bin/systemd-nspawn --quiet --keep-unit --boot --network-bridge=${BRIDGE} --machine=nsdc ${OPTIONS}
+ExecReload=/usr/bin/systemctl -M nsdc daemon-reload
```

Comment:
The `daemon-reload` command seems synchronous: it exists when the new configuration has been applied